src/utils/Configuration.ts

```diff
@@ -178,6 +180,19 @@ export default class Configuration {
     return ocpiService;
   }
 
+  // OICP Server Configuration
+  public static getOICPServiceConfig(): OICPServiceConfiguration {
+    const oicpService = Configuration.getConfig().OICPService;
```

Comment:
Security issue: Key and Cert would have to be provided by the User Defined Service in SCP.
You can check this with Jerome.